app/subscriber/src/features/my-reports/admin/ReportAdminSettings.tsx

```diff
@@ -44,7 +44,12 @@ export const ReportAdminSettings: React.FC = () => {
   return (
     <Col gap="1rem">
       <Box title="Identify your report" icon={<FaFileInvoice />}>
-        <FormikText name="name" label="Report Name:" required />
+        <FormikText
+          name="name"
+          label="Report Name:"
+          required
+          placeholder="Enter unique report name"
```

Comment:
The error that occurs when a report has the same name isn't great.  But this may reduce that type of issue.